source/includes/compatibility-table.rst

```diff
@@ -0,0 +1,11 @@
+.. list-table::
+   :header-rows: 1
+   :widths: 50 25 25
+
+   * - {+provider-long+}
+     - Entity Framework Core
+     - .NET
+
+   * - **{+version-number+}**
+     - **7.0**
```

Comment:
S: We have a constant for framework version (7.0). Not sure whether both are needed if the docs are tied to the framework version. Maybe not worth it to use here since we'd just have to change it next version. Nevermind!